MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationComponentActivationOptions.java

```diff
@@ -208,6 +233,28 @@ public Builder useDefaultLocationEngine(boolean useDefaultLocationEngine) {
       return this;
     }
 
+    /**
+     * True if you want to initialize and use the specialized location layer to render the location puck.
+     * <p>
+     * The specialized layer comes with significant performance improvements,
+     * but since it's not based on the runtime styling,
+     * it's not fully compatible with the traditional implementation. The incompatibilities are:
+     * <ul>
+     * <li> Constants like {@link LocationComponentConstants#FOREGROUND_LAYER}
+     * or {@link LocationComponentConstants#LOCATION_SOURCE} are ignored.
+     * <li> All options that alter the image ID, like {@link LocationComponentOptions#foregroundName()}, are ignored.
+     * Use {@link LocationComponentOptions#foregroundDrawable()} instead.
+     * </ul>
+     *
+     * @param useSpecializedLocationLayer true if you want to initialize and use the
+     *                                    specialized location layer. Defaults to false.
+     * @return the {@link Builder} object being constructed
+     */
+    public Builder useSpecializedLocationLayer(boolean useSpecializedLocationLayer) {
+      this.useSpecializedLocationLayer = useSpecializedLocationLayer;
+      return this;
+    }
```

Comment:
@chloekraw I'd love your review on the documentation here.

Reply:
Added two suggestions -- otherwise, looks good! Thanks @LukasPaczos.